### DISCUSSION TITLE
Which module is missing?

### DISCUSSION THREAD

**Discussion by puneet-vmware** [vmware/clarity#5961] [Q&A] [answered]
Angular and cli version: 12.0.0
> node --version v12.22.1
> npm --version 6.14.12

> ng new my-awesome-app
> cd my-awesome-app
> ng serve
[works]

> ng add @clr/angular
> ng serve

Error: Module not found: Error: Can't resolve '/home/ubuntu/my-awesome-app/node_modules/@babel/runtime/helpers/esm/asyncToGenerator' in '/home/ubuntu/my-awesome-app/node_modules/@cds/core/internal/motion'
Did you mean 'asyncToGenerator.js'?

**Comment by bbogdanov** [comment, marked as answer]
Hi @puneet-vmware, We are working on providing support for Angular 12. It will be available soon.

**Reply by puneet-vmware** [reply]
Where can I find the supported list of pr-req version 
eg: Angular, node, npm, etc?

**Reply by puneet-vmware** [reply]
found Angular 11.2.5 from 5.2.0 release notes. Downdraded Angular to 11.2.5 and its working now! Thanks :)
